7/paul-battley-captions.srt

```diff
@@ -4353,7 +4354,7 @@ And now I should have... 42
 1023
 01:33:50,008 --> 01:33:55,576
 Yes, it definitely doesn't work
-because I say def initialize with a node.
+because I… say def initialize with a node.
```

Comment:
We usually use three periods for elipsis but this unicode elipsis character looks good!